packages/atlas-experiment-table/html/index-sc.html

```diff
@@ -88,7 +104,7 @@
             width: 0.5
           },
           dataRows: data,
-          host: 'https://www.ebi.ac.uk/gxa/sc/'
+          host: 'http://localhost:8080/gxa/sc/'
```

Comment:
I guess it is better to keep it `https://www.ebi.ac.uk/gxa/sc`. I understand that your backend changes might not have been merged yet but for future purposes, it would be better.

Reply:
Restored to `www.eb.ac.uk`.